content/docs/atom-guide/features/taa.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Temporal Anti-Aliasing
+description: "Learn about temporal anti-aliasing in the Atom Renderer"
+toc: true
+---  
+
+*Temporal anti-aliasing* (TAA) is a type of supersampling that takes samples from different locations within a pixel each frame and combines them with samples from previous frames to create a supersampled image. The main challenge of TAA is knowing when samples from the previous frame are valid. TAA may cause some softness in your image, especially while in motion. To improve the quality of your image, you can enable [contrast adaptive sharpening](cas/), which sharpens the details of your image. 
+
+
+## Parameters
+
+| Parameter | Description | Default |
+| - | - | - |
+| **m_currentFrameContribution** | Controls how much the current frame's samples contribute to the final image. Reducing this value can help prevent flickering in some circumstances. | `0.1` |
+| **m_clampGamma** | Controls the number of standard deviations from the mean of the 3x3 current neighborhood that the history is clipped to. Increasing this value may prevent flickering, but can also lead to ghosting. | `1.0` |
+| **m_maxDeviationBeforeDampening** | Controls the number of standard deviations from the mean, which defines a threshold for the pixel's brightness before it begins to reduce its contribution. This prevents flickering by lowering the weight of the current pixel, if its brightness is very different from the surrounding neighborhood and history. By default, any pixel with a brightness that is more than 0.5 standard deviations from the mean will begin to reduce its contribution. | `0.5` |
+
+
+## Using TAA
+
+You can enable `Taa.pass` in the main render pipeline through the parent pass, `PostProcessParent.pass`. By default, TAA is disabled. To enable it, set `Enabled` to `true` in the parent pass.
+
+You can also add `Taa.pass` to a custom pipeline. 
+
+
+## Issues and improvements
+
+The following are a list of known issues and improvements for TAA:
+
+- The bloom effect causes flickering when TAA is enabled. 
+  
+- Aux-Geom causes jittering because it uses the jittered view, but applies it after TAA runs. Therefore, TAA currently is not recommended to use in the **O3DE Editor**.
+
+- Motion vectors on parallel occlusion mapping are only accurate when pixel depth offset is enabled. However, this can lead to additional blur when in motion.
```

Comment:
The way this is worded is a little backwards. Motion vectors on POM won't be accurate unless PDO is enabled - the greater the depth offset from pom, the more inaccurate the motion vectors may become. Inaccuracies in the motion vectors is what can lead to additional blur (only when the object or camera is in motion though). If PDO is enabled this problem goes away, but PDO is expensive so that's not really recommended. Usually standard history rejection and rectification make this problem hard to notice, but it depends a lot on the material and mesh, so ymmv.